src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -154,5 +155,22 @@ def add_suggestion(kw, suggestion):
             " be selected on request from the GUI, and there is also an alternative"
             " view accessible by using ert viz in the commandline.",
         )
+        add_suggestion(
+            "UPDATE_PATH",
+            "The UPDATE_PATH keyword has been removed and no longer has any effect. "
+            "It can be safely removed. It has been used in the path to set different "
+            "python versions for the forward model. This should no longer be necessary,"
+            " and could potentially cause trouble."
+            "\n\n"
+            "If you really need to update a path on the compute node, you can use the "
+            "SETENV keyword to achieve the same. Feel free to contact the ERT community"
+            " for support!"
+            "\n\n"
+            "For updating a path with SETENV you would write something like:"
+            "\n\n"
+            "SETENV MY_PATH /added/path:$MY_PATH"
```

Comment:
Likely this should not have to be added so I suggest we remove it and instead give a link to the docs.